js/controller.js

```diff
@@ -266,6 +266,12 @@
                 $scope.focus = "dilbert";
             });
 
+            //show hourly weather
+            AnnyangService.addCommand('Show (the) weather', function() {
```

Comment:
Should be
``` javascript
AnnyangService.addCommand('Show (me)(the) weather', function() {
```
Which will match
- show weather
- show me weather
- show the weather
- show me the weather